src/target/spirv/ir_builder.h

```diff
@@ -499,7 +499,8 @@ class IRBuilder {
    * \param binding The binding locaiton in descriptor set
```

Comment:
Please add the description of the new param `descriptor_set`, thank you!

Reply:
Thank you, and added.